Makefile

```diff
@@ -334,7 +342,8 @@ $(INSTALLER_IMG): PKG=mkimage-raw-efi
 $(KERNEL_IMG): PKG=new-kernel
 $(IPXE_IMG): PKG=ipxe
 $(BIOS_IMG): PKG=uefi
-$(EFI_PART) $(BOOT_PART) $(INITRD_IMG) $(INSTALLER_IMG) $(KERNEL_IMG) $(IPXE_IMG) $(BIOS_IMG): $(LINUXKIT) | $(INSTALLER)
+$(UBOOT_IMG): PKG=u-boot
+$(EFI_PART) $(BOOT_PART) $(INITRD_IMG) $(INSTALLER_IMG) $(KERNEL_IMG) $(IPXE_IMG) $(BIOS_IMG) $(UBOOT_IMG): $(LINUXKIT) | $(INSTALLER)
```

Comment:
UBOOT_IMG only applies for riscv but it is always set and used here. Is that a problem?

Reply:
What is set here is a recipe on how to get the u-boot binary from our packages -- that binary exists on every single architecture we support. So in that sense -- this is just a generic rule @eriknordmark